src/modal/modal-container.component.ts

```diff
@@ -18,7 +18,7 @@ import { isBs3 } from '../utils/theme-provider';
 @Component({
   selector: 'modal-container',
   template: `
-    <div [class]="'modal-dialog' + (config.class ? ' ' + config.class : '')" role="document">
+    <div [class]="'modal-dialog' + (config.class ? ' ' + config.class : '')">
```

Comment:
why did you remove `role="document"?` original bootstrap' modal has it https://getbootstrap.com/docs/4.0/components/modal/#modal-components